docs/components/card.md

```diff
@@ -239,19 +240,24 @@ export interface Tooltip {
 }
 ```
 
+### Spacing
+
+Same as, `<SCardBlock>`, `<SCardFooter>` also comes with `:space` props that lets you control the padding of the block. You may pass either `compact` or `wide` as a value.
+
+```ts
+interface Props {
+  space?: 'compact' | 'wide'
+}
+```
+
 ```vue-html
 <SCard>
-  <SCardFooterActions>
-    <SCardFooterAction
-      mode="mute"
-      label="Cancel"
-      @click="onClick"
-    />
-    <SCardFooterAction
-      mode="info"
-      label="Submit"
-      @click="onClick"
-    />
+  <SCardFooter space="compact">
+    ...
   </SCardFooter>
 </SCard>
 ```
+
+To ensure consistent spacing across the application, you should adjust the spacing based on the size of the `<SCard>` component. Consider using the `compact` class for card widths smaller than `640px` and the `wide` class for larger widths.
```

Comment:
Since these are props, not classes, I think _setting_ or _value_ makes sense here 😉 

```suggestion
To ensure consistent spacing across the application, you should adjust the `space` setting based on the size of the `<SCard>` component. Consider using the `compact` setting for card widths smaller than `640px` and the `wide` setting for larger widths.
```